lib/backend/k8s/k8s.go

```diff
@@ -149,6 +157,7 @@ func (c *KubeClient) ensureThirdPartyResources() error {
 // do not already exist.
 func (c *KubeClient) createThirdPartyResources() (bool, error) {
```

Comment:
It's so we can use the wait.PollImmediate above, but you're right we get all the information we need from the error, so we can simplify.